meson.build

```diff
@@ -2294,6 +2295,19 @@ pam_template = {
                 libshared_static,
         ],
         'dependencies' : [
+                # Note: our PAM modules also call dlopen_libpam() and use
+                # symbols aquired through that, hence the explicit dep here is
```

Comment:
aquired -> acquired

Reply:
typo fixed with a direct push to git btw